Description: Add Makefile and generate.pe for font generation
 This patch adds the Makefile and generate.pe which is used for
 generating the font from source.
Author: Vasudev Kamath <kamathvasudev@gmail.com>
Forwarded: no
Last-Update: 2020-04-27

--- /dev/null
+++ b/generate.pe
@@ -0,0 +1,19 @@
+#!/usr/bin/env fontforge
+
+# script file for FontForge for TTF generation
+# usage:
+#   mkdir TTFfiles
+#   chmod +x generate.pe
+#   ./generate.pe *.sfd
+
+if ($version < "20061220")
+  Error("Your version of FontForge is too old - 20061220 or newer is required");
+endif
+i = 1
+while ( i < $argc )
+  Open($argv[i], 1)
+  #Use gen_flags=Defualt fmflags value as -1
+  gen_flags = -1
+  Generate("" + $fontname + ".ttf", "", gen_flags)
+  i++
+endloop
--- /dev/null
+++ b/Makefile
@@ -0,0 +1,3 @@
+default:
+	chmod +x generate.pe
+	./generate.pe *.sfd
